root/Application.cfc

```diff
@@ -1,11 +1,31 @@
 component {
 
-	this.name = "lucee-provider";
-	
-	this.s3.accessKeyId = server.system.environment.S3_EXTENSION_ACCESS_KEY_ID;
+	this.name            = "lucee-provider";
+	this.s3.accessKeyId  = server.system.environment.S3_EXTENSION_ACCESS_KEY_ID;
 	this.s3.awsSecretKey = server.system.environment.S3_EXTENSION_SECRET_KEY;
 
-	if ( left( cgi.script_name, 5 ) neq "/rest" ) {
-		location url="https://www.lucee.org" addtoken=false;
+	function onApplicationStart() {
+		application.cdnUrl = server.system.environment.S3_CDN_URL ?: "https://ext.lucee.org/";
```

Comment:
there are two CDNs, this is used for both providers?
- `cdn.lucee.org` for lucee versions 
- `ext.lucee.org` for extensions